### PR TITLE
192 met and axway

### PR DESCRIPTION
**Pull request by SteveScott**
#192 Changed the folder. Need to change the credentials in the Github secrets.

### CODE REVIEW

File: bin/axway.sh [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
 source $(pwd)/bin/cli.sh
 
+#Steve added for error catching
+set -o pipefail  # trace ERR through pipes
+set -o errtrace  # trace ERR through 'time command' and other functions
+#set -o nounset   ## set -u : exit the script if you try to use an uninitialised variable
+set -o errexit   ## set -e : exit the script if any statement returns a non-true return value
+
 function axway_ls {
-    sftp -q -i ~/.ssh/id_rsa_axway \
```
Comment (YuliaZamriy) [inline bin/axway.sh:5]: Not adding -v here?

File: bin/axway.sh [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
 EOF
 }
 register 'axway' 'ls' 'listing directory' axway_ls
 
 
 function axway_cmd {
-    sftp -q -i ~/.ssh/id_rsa_axway \
-    -o HostKeyAlgorithms=+ssh-dss \
-    -o KexAlgorithms=diffie-hellman-group14-sha1 \
+    sftp -i ~/.ssh/id_rsa_axway \
```
Comment (YuliaZamriy) [inline bin/axway.sh:22]: Not adding -v here?

File: recipes/met/runner.sh [hunk 3]
```diff
@@ -12,7 +12,7 @@ ACL=private
     rm -rf output && mkdir -p output
 
 
-    files=$(axway_ls -nrt Met_Museum/Attendance | grep .csv | awk '{print $NF}')
+    files=$(axway_ls -nrt Met_Museum_2/Attendance | grep .csv | awk '{print $NF}')
```
Comment (YuliaZamriy) [inline recipes/met/runner.sh:15]: Full folder name on Axway is "Met_Museum_2 (NYCAnalytics@cityhall.nyc.gov)"

File: recipes/met/runner.sh [hunk 4]
```diff
@@ -24,7 +24,7 @@ ACL=private
     done
     
 
-    files=$(axway_ls -nrt Met_Museum/Membership | grep .csv | awk '{print $NF}')
+    files=$(axway_ls -nrt Met_Museum_2/Membership | grep .csv | awk '{print $NF}')
```
Comment (YuliaZamriy) [inline recipes/met/runner.sh:27]: Full folder name on Axway is "Met_Museum_2 (NYCAnalytics@cityhall.nyc.gov)"

File: recipes/met/runner.sh (hunk 3 above)
Comment (SteveScott) [reply recipes/met/runner.sh:15]: @YuliaZamriy there are two folders in axway. One with and one without the email address. Why are there two?

File: bin/axway.sh (hunk 2 above)
Comment (SteveScott) [reply bin/axway.sh:22]: the Verbose is very noisy and has sensitive information. It is more for debugging. We know it is connecting now so we don't need to turn on verbose. I did turn off quiet though.

Comment (YuliaZamriy) [reply bin/axway.sh:22]: Ok.

File: recipes/met/runner.sh (hunk 3 above)
Comment (YuliaZamriy) [reply recipes/met/runner.sh]: @SteveScott That's a very good question. We should ask DOITT about that. But Met_Museum_2 doesn't have any data, while "Met_Museum_2 (NYCAnalytics@cityhall.nyc.gov)" has the most recent datasets.

**Pull request merged**